server/IdentityLocalState.ts

```diff
@@ -0,0 +1,24 @@
+import type { Response } from 'express';
+
+//could this be unified with IdentityDetails in globalstate.ts ?
```

Comment:
I didn't mean to include this comment.  I decided against doing this at least in this PR because a lot of things are using res.locals.identity.  I would have to make sure nothing is using the old name  and  typescript won't help me here (as usual).
I can always do a text search and replace like a cowboy (or ever more reckless, a js developer)